oda_api/data_products.py

```diff
@@ -610,7 +610,12 @@ def __init__(self,cat_dict,name='catalog'):
         if 'cat_coord_units' in cat_dict.keys():
             coord_units = cat_dict['cat_coord_units']
 
-        meta = {'FRAME': frame}
+        if 'cat_meta' in cat_dict.keys():
+            meta = cat_dict['cat_meta']
+        else:
+            meta = {}
```

Comment:
What is `cat_dict['cat_meta']` supposed to contain? Is it an dict entry? 

Cause otherwise I would use the same approach of the other if branches of this function, and init `meta` at the top of the function.

Reply:
Yes, this is a dict containing catalog table metadata. It is attached to the catalog table in the backend (6th cell of https://github.com/oda-hub/gw-backend/blob/master/nb/conesearch.ipynb) and passed as-is through the plugin https://github.com/oda-hub/dispatcher-plugin-gw/blob/142a37cf8ee0e5527aa15dccdbd51920a3451d6c/dispatcher_plugin_gw/products.py#L277

I think that this way is rather universal, we don't need to know, which fields are there.

Reply:
ok then I would just suggest the following to have a more consistent code:

```suggestion
        if 'cat_meta' in cat_dict.keys():
            cat_meta_entry = cat_dict['cat_meta']
            
        meta.update(cat_meta_entry)
```

and have `meta = {}` at the top of the method